source/changelogs/2021-09-01-September.md

```diff
@@ -0,0 +1,34 @@
+---
+title: September 2021 Changelog
+changelog: true
+description: September 2021 Pantheon changelog
+reviewed: "2021-09-01"
+---
+
+## Platform Improvements
+
+### Drupal 9.2.3
+
+[Drupal 9.2.3](https://www.drupal.org/project/drupal/releases/9.2.3) is now available for production sites. [Drupal 9](/drupal-9) sites on Pantheon use [Integrated Composer](/integrated-composer), enabling one-click core updates through the Dashboard. To check for available updates, click **Check Now** from the Site Dashboard **Dev** tab.
+
+Drupal 9.2.x will receive security coverage until June 15, 2022. Sites on 9.1.x or earlier should update to [Drupal 9.1.11](https://www.drupal.org/project/drupal/releases/9.1.11).
+
+
+### Drupal 8.9.18
+
+[Drupal 8.9.18](https://www.drupal.org/project/drupal/releases/8.9.18) is now available on the Pantheon platform. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates).
+
+
+### Drupal 7.82
+
+[Drupal 7.82](https://www.drupal.org/project/drupal/releases/7.82) is now available on the Pantheon platform. Pantheon is committed to providing Drupal 7 extended support for all customers through 2025, past Drupal 7’s end of life in 2022.
+
+
+
+## Documentation
+
+[Autopilot](/guides/autopilot) - The Autopilot guide has been updated now that Autopilot is available for all users with Gold Accounts and higher.
+
+[Pantheon Disaster Recovery Playbook](/guides/disaster-recovery) - Address and troubleshoot emergency downtime situations on the Pantheon platform. The guide focuses on the immediate actions a Pantheon customer should take in the event of a catastrophic site failure. 
```

Comment:
Disaster Recovery is now called Multizone Failover. That guide should be renamed before this goes out:
- https://pantheon.io/docs/multizone-failover
- https://pantheon.io/features/multizone-failover

Reply:
@joa-pan clarified - this isn't about the feature, it's about what to do. I misunderstood - LGTM